### ISSUE TITLE
No longer being worked on -- looking for maintainer

### ISSUE THREAD

**Issue by ppKrauss** [meteor/postgres-packages#65]
Was an "in development" project, but ~16 months with no news... The status now is "abandoned"?

And about new features of [JSON and JSONb  functions and operators of Pg v9.6+](https://www.postgresql.org/docs/current/static/functions-json.html)?  For "read-only" operations the JSON-algebra is so perfect as MongDB query operations!

**Comment by stubailo** [comment]
That's correct, we are no longer working on this. I'd be happy to hand over the project if someone is interested. After investigating how to abstract out the data system we decided GraphQL was a better solution and are now working on GraphQL tools: http://dev.apollodata.com/
